spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -471,7 +481,7 @@ def get_data_or_scalar(argument, data_in):
         # build data2 with same shape as data
         data_out = data_in[:, :, :] * 0 + float(argument[0])
```

Comment:
At first glance I thought that `data_in[:, :, :]` would have the effect of truncating `data_in` to 3 dimensions, but a little testing at the REPL shows me that it doesn't: you get an IndexError if `data_in` has fewer than 3 dimensions, but you get an _exact_ copy of `data_in` (of the same shape) if it has 3 _or more_ dimensions. So, `data_in[:, :, :] * 0` is mostly just a strange and possibly misleading spelling of `np.zeros_like(data_in)`.

Also, the whole line could be replaced by `data_out = np.full_like(data_in, float(argument[0]))`.